doctr/models/detection/linknet/pytorch.py

```diff
@@ -177,22 +177,27 @@ def compute_loss(
 
         seg_target, seg_mask = torch.from_numpy(seg_target).to(dtype=out_map.dtype), torch.from_numpy(seg_mask)
         seg_target, seg_mask = seg_target.to(out_map.device), seg_mask.to(out_map.device)
-        proba_map = torch.sigmoid(out_map)[seg_mask]
+        seg_mask = seg_mask.to(dtype=torch.float32)
 
-        # Focal loss
         bce_loss = F.binary_cross_entropy_with_logits(out_map, seg_target, reduction='none')
```

Comment:
That's where alpha intervenes:
```python
bce_loss = F.binary_cross_entropy_with_logits(out_map, seg_target, reduction='none', pos_weight=alpha * torch.ones(out_map.shape[1], dtype=torch.float32, device=out_map.device)))
```

Alpha is the positive weight term in the loss (cf. https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html)

Reply:
For the TF implementation though, there isn't such parameter